File: integration-tests/freemarker/src/main/java/io/quarkus/it/freemarker/Person.java
package io.quarkus.it.freemarker;

import io.quarkus.runtime.annotations.RegisterForReflection;

@RegisterForReflection
public class Person {

    public String name;

    public Person setName(String name) {
        this.name = name;
        return this;
    }
}

File: integration-tests/kotlin-serialization/src/main/kotlin/io/quarkus/it/kotser/model/Person.kt
package io.quarkus.it.kotser.model

import io.quarkus.runtime.annotations.RegisterForReflection
import kotlinx.serialization.Serializable

@Serializable
@RegisterForReflection
data class Person(var name: String, var defaulted: String = "hi there!") {
    override fun toString(): String {
        TODO("this shouldn't get called. a proper serialization should be invoked.")
    }
}
